src/main/java/examples/Examples.java

```diff
@@ -803,4 +803,25 @@ public void collector02Example(PgClient client) {
         }
       });
   }
+
+  public void cancelRequest(PgConnection connection) {
+    connection.query("SELECT pg_sleep(20)", ar -> {
```

Comment:
20 seconds or millis ?

Reply:
20 seconds here, maybe it could be shorter

Reply:
it's fine it is example :-)